linkml/generators/shaclgen.py

```diff
@@ -55,13 +56,15 @@ def as_graph(self) -> Graph:
         for pfx in self.schema.prefixes.values():
             g.bind(str(pfx.prefix_prefix), pfx.prefix_reference)
 
+        print(f"# class suffix: {self.suffix}")
```

Comment:
my guess on the docs failing would be this print statement being left in messing up the detection of headers and transitions when sphinx goes to do autoclass at the bottom. that is a pretty cryptic error, haven't found a better way to debug it.

Reply:
Removed that print but it is still failing.
My next try would be moving @click.command above @click.option. I have read that command should encapsulate the option. Let's see.

Reply:
Still failing. But now it fails in line 20 (it was 18 before):
/home/runner/work/linkml/linkml/docs/generators/shacl.rst:20: CRITICAL: Unexpected section title or transition.
Need to think about next steps.